docs/site/pages/index.html

```diff
@@ -62,7 +62,7 @@ <h1 class="product-intro__title text_lead">
           </svg>
         </div>
         <div class="product-intro__certificate--text">
-          <span>Certified in CNCF</span>for Kubernetes 1.19—1.23
+          <span>Certified in CNCF</span>for Kubernetes 1.20—1.23
```

Comment:
@nabokihms Would it be better to leave it as it is?

Reply:
rollbacked